### PR TITLE
fix humanoid pick place animation

### PR DESCRIPTION
**Pull request by jimmytyyang**

### CODE REVIEW

**Review by xavierpuigf — APPROVED** [review]
LGTM

File: habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py [hunk 1]
```diff
@@ -326,25 +326,39 @@ def comp_values(self, index):
...
+            value = min(value, maxv)
             value_norm = (value - minv) / (maxv - minv)
 
             index = value_norm * (num_bins - 1)
             # lower = max(0, math.floor(index))
             # upper = min(math.ceil(index), num_bins - 1)
 
-            lower = min(max(0, math.floor(index)), num_bins - 1)
+            lower2 = min(math.floor(index), num_bins - 1)
+            # lower = min(max(0, math.floor(index)), num_bins - 1)
```
Comment (0mdc) [inline habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py:345]: We can probably remove this line.

File: habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py [hunk 2]
```diff
@@ -375,16 +389,16 @@ def inter_data(x_i, y_i, z_i, dat, is_quat=False):
...
+            #     c10 = normalize_quat(c10)
+            #     c11 = normalize_quat(c11)
             c0 = c00 * (1 - yd) + c10 * yd
             c1 = c01 * (1 - yd) + c11 * yd
-            if is_quat:
-                c0 = normalize_quat(c0)
-                c1 = normalize_quat(c1)
+            # if is_quat:
+            #     c0 = normalize_quat(c0)
+            #     c1 = normalize_quat(c1)
```
Comment (0mdc) [inline habitat-lab/habitat/articulated_agent_controllers/humanoid_rearrange_controller.py:401]: Do we still need those comments?

**Review by 0mdc — APPROVED** [review]
LGTM! I'll let you see if we can clean up the comments I highlighted.

**Pull request merged**